pkg/reconciler/autoscaling/kpa/scaler_test.go

```diff
@@ -88,6 +89,27 @@ func TestScaler(t *testing.T) {
 			kpaMarkActive(k, time.Now().Add(-stableWindow).Add(1*time.Second))
 		},
 		wantCBCount: 1,
+	}, {
+		label:         "waits to scale to zero (just before idle period), custom PA window",
+		startReplicas: 1,
+		scaleTo:       0,
+		wantReplicas:  1,
+		wantScaling:   false,
+		kpaMutation: func(k *pav1alpha1.PodAutoscaler) {
+			WithWindowAnnotation(paStableWindow.String())(k)
+			kpaMarkActive(k, time.Now().Add(-paStableWindow).Add(1*time.Second))
+		},
+		wantCBCount: 1,
+	}, {
+		label:         "custom PA window, check for standard window, no probe",
```

Comment:
nit: describe that the custom PA window is smaller than standard window?

Reply:
added the comment